pkg/parca/parca.go

```diff
@@ -133,6 +135,11 @@ func Run(ctx context.Context, logger log.Logger, reg *prometheus.Registry, flags
 		defer closer()
 	}
 
+	if flags.Port != "" {
+		level.Warn(logger).Log("msg", "flag --port is deprecated, use --http-address instead")
```

Comment:
We broke a bunch of flags in this release I think adding one more doesn’t make a difference

Reply:
Agreed.

Reply:
👍

Reply:
I'd done it but then reverted; it broke the snap. I think this is one of the oldest flags so let's don't break the backwards compatibility on this one.